tests/test_spacy_classifier.py

```diff
@@ -45,3 +45,25 @@ def test_multilabel_Y_list():
     model.fit(X, Y)
     assert model.score(X, Y) > 0.4
     assert model.predict(X).shape == (5,4)
+
+def test_partial_fit():
+    X = [
+        "One and two",
+        "One only",
+        "Three and four, nothing else",
+        "Two nothing else",
+        "Two and three"
+    ]
+    Y = [
+        [1,1,0,0],
+        [1,0,0,0],
+        [0,0,1,1],
+        [0,1,0,0],
+        [0,1,1,0]
+    ]
+
+    model = SpacyClassifier()
+    for x, y in zip(X, Y):
+        model.partial_fit([x], [y])
+    assert model.score(X, Y) > 0.4
```

Comment:
We could lower this to 0.3, or even just `model.score(X, Y) > 0` to ensure the pipeline passes? I think this is trying to test whether the code runs correctly and output a non-zero number, which probably means that it is training.

Reply:
I think we need to make sure it learns something better than chance. Reducing to 0.3 for now.